pkg/server/apiextensions.go

```diff
@@ -65,18 +66,25 @@ func (c *inheritanceCRDLister) ListWithContext(ctx context.Context, selector lab
 	}
 
 	// Check for API inheritance
+	const adminClusterName = "admin"
```

Comment:
(No action required) For future, maybe reading the cluster/clusterName/workspace from context could be factored out into a common method?